plop-templates/introToStories.stories.mdx.hbs

```diff
@@ -0,0 +1,126 @@
+import { Meta } from '@storybook/addon-docs';
```

Comment:
Question; do we need to duplicate this file for each component or should there be one `introToStories` in the project?

Reply:
i removed duplicates :)